src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+import java.util.Scanner;
+
 public class Duke {
+    static int noOfTasks = 0;
+    static String sectionDivider = "____________________________________________________________";
```

Comment:
I really like the use of camelCase for your variable names, but I am concerned that the name of the counter variable noOfTasks might be confusing as number was abbreviated to no. Maybe you could consider spelling number out?